fulvio-almeida/1.i/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sistemas de Informação</title>
+
+    <link type="text/css" href="style.css">
```

Comment:
It is missing the attribute `rel` to inform the relationship of this link.

![image](https://user-images.githubusercontent.com/5324414/27113448-3953ca18-5093-11e7-98bf-c4bc8a32461a.png)

https://developer.mozilla.org/en/docs/Web/HTML/Element/link